test/unit/Git/CommitFileViaConsoleTest.php

```diff
@@ -93,6 +106,12 @@ public function testFailsIfNotOnCorrectBranch(): void
         $this->expectException(InvalidArgumentException::class);
         $this->expectExceptionMessage('different branch');
         (new CommitFileViaConsole())
-            ->__invoke($this->checkout, BranchName::fromName('1.0.x'), 'README.md', 'commit message');
+            ->__invoke(
```

Comment:
I think we meed a test that verifies that the commit is signed

Reply:
Assertion added!